doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Sonata project contribution
+
+Thanks for you interest onto Sonata projects!
+
+## Summary
+
+* [Issues]()
+* [Pull Requests](#pull-requests)
+* [Label rules]()
+
+## Pull Requests
+
+All the sonata team will be glad to review your code changes propositions! :smile:
+
+But please, read the following before.
+
+### Coding style
+
+Each project follows [PSR-1](http://www.php-fig.org/psr/psr-1/), [PSR-2](http://www.php-fig.org/psr/psr-2/)
+and [Symfony Coding Standards](http://symfony.com/doc/current/contributing/code/standards.html) for coding style,
+[PSR-4](http://www.php-fig.org/psr/psr-4/) for autoloading.
+
+Please [install PHP Coding Standard Fixer](http://cs.sensiolabs.org/#installation)
+and run this command before committing your modifications:
+
+```bash
+php-cs-fixer fix --verbose
+```
+
+### Writing a Pull Requests
```

Comment:
Since GitHub has the "Squash and merge" functionality, it's not needed to ask contributors to squash the history.

You only have to do this:

![image](https://cloud.githubusercontent.com/assets/1698357/14673583/11d5fe40-06ff-11e6-8636-e682234040a7.png)

On the other side, we should add a Team documentation to how to merge a PR, introducing this method.